### PR TITLE
Update rubygem-hammer_cli_import to 0.11.3

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Review by stbenjam — CHANGES_REQUESTED** [review]
Please update the RPM spec to match the gemspec in terms of dependencies, that's why this happened to begin with.

If the deps were right, repoclosure would've failed when hammer_cli_katello was released and we'd have known sooner

**Comment by ehelms** [comment]
@stbenjam updated the spec file

File: rubygem-hammer_cli_import/rubygem-hammer_cli_import.spec [hunk 1]
```diff
@@ -25,7 +25,8 @@ Obsoletes: rubygem-%{gem_name} < 0.10.21-3
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix}rubygem(hammer_cli) >= 0.1.3
```
Comment (stbenjam) [inline rubygem-hammer_cli_import/rubygem-hammer_cli_import.spec]: There's no minimum in the gemspec

**Review by stbenjam — CHANGES_REQUESTED** [review]

File: rubygem-hammer_cli_import/rubygem-hammer_cli_import.spec (hunk 1 above)
Comment (ehelms) [reply rubygem-hammer_cli_import/rubygem-hammer_cli_import.spec]: Good call - updated

**Review by stbenjam — APPROVED** [review]

**Comment by stbenjam** [comment]
ack!

**Pull request merged**